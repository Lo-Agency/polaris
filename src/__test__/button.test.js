import React from 'react';
import { render } from '@testing-library/react';

import Button from '../components/atoms/button';

describe('Button component', () => {
	it('it should render snapshot button component', () => {
		const { asFragment } = render(<Button />);
		expect(asFragment(<Button />)).toMatchSnapshot();
	});
});
