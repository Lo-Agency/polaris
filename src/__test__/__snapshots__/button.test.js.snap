// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component it should render snapshot button component 1`] = `
<DocumentFragment>
  <button
    type="submit"
  />
</DocumentFragment>
`;
